tests/backends/postgresql/test_server_side_cursors.py

```diff
@@ -80,3 +80,20 @@ def test_server_side_cursors_setting(self):
             next(persons)  # Should not open a server-side cursor
             cursors = self.inspect_cursors()
             self.assertEqual(len(cursors), 0)
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', "Test only for PostgreSQL")
+class DatabaseSequenceTests(TestCase):
+    def test_sequence_introspection(self):
```

Comment:
test_get_sequences? Should other backends have similar tests? (e.g. Oracle also returns 'name')